presto/eval/eurosat_eval.py

```diff
@@ -152,12 +157,11 @@ def __init__(
         assert IMAGE_SIZE % input_patch_size == 0
         self.num_patches_per_dim = int(IMAGE_SIZE / input_patch_size)
         self.num_patches = self.num_patches_per_dim**2
+        self.batch_size = min(self.num_patches * 64, 40920)
```

Comment:
Still confusing to me why you have `64` here and the same line but with `self.num_patches * 36` in TreeSatEval